Webinar-12-12-2018/TukeyBoxPlotsTestv5.R

```diff
@@ -0,0 +1,65 @@
+rm(list=ls())  ## remove old objects
```

Comment:
I'd take this off (I know it's Paul's script, but to reduce the chance someone else will use it). It's bad practice to include this in a script that is being shared. I was thoroughly scolded when I started using R when I shared a script like that. I now generally only passive-aggressively remove the line from other people's code a la:

https://twitter.com/sharlagelfand/status/1072307346959990784

😸